pkg/cli/changes.go

```diff
@@ -49,7 +49,12 @@ func getGetChangesCommand() *cobra.Command {
 }
 
 func runChangesCommand(cmd *cobra.Command, args []string) error {
-	client := diags.NewConfigDiagsClient(getConnection())
+	clientConnection, clientConnectionError := getConnection()
```

Comment:
FWIW the idiomatic Go naming pattern would have this line as:
```go
conn, err := getConnection()
if err != nil {
    return err
}
```

Reply:
Also FWIW this is contrary to the classic literature on writing clean and maintainable code. 🤷‍♂